### DISCUSSION TITLE
how to close all modals using $dispatch('closeModal')

### DISCUSSION THREAD

**Discussion by khalidmaquilang** [wire-elements/modal#444] [Q&A] [answered]
I have parent modal and a child.. once I clicked the close buttom from the child which triggers $dispatch('closeModal').. it should also close the parent

**Comment by PhiloNL** [comment, marked as answer]
`$dispatch('closeModal', {force: true})`

**Reply by khalidmaquilang** [reply]
ohh thank you for this! 

i thought it is not working but i checked my code and i used ``$dispatch('closeModal', true)``
